jre_emul/android/platform/libcore/harmony-tests/src/test/java/org/apache/harmony/tests/java/util/zip/ZipEntryTest.java

```diff
@@ -20,13 +20,22 @@
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
+//import java.nio.file.attribute.FileTime;
```

Comment:
Probably we need to add a "J2ObjC removed" comment above and follow the convention.

Reply:
added J2ObjC comments